jsonify.js

```diff
@@ -27,8 +26,7 @@ var lf_processArguments = function(){
 
     while (0 < l_aryArgs.length){
         l_strFlag = l_aryArgs.shift();
-        console.log(l_strFlag);
-        console.log(l_regexType.test(l_strFlag));
+
         if(l_regexDirectory.test(l_strFlag)){
```

Comment:
Have you considered using a switch case here? Might be a good candidate to make it easier to read and comment each branch.